src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -245,6 +248,67 @@ void SetTableValue(TableValueBuilder tbv)
             };
         }
 
+        private static string GuessContentTypeFromName(string name)
```

Comment:
Move this to the studio, see: https://developer.mozilla.org/en-US/docs/Web/API/Blob/type